code/modules/mod/modules/_module.dm

```diff
@@ -258,10 +259,14 @@
 /obj/item/mod/module/proc/on_device_deletion(datum/source)
 	SIGNAL_HANDLER
 
+	if(QDELETED(src))
+		return
+
```

Comment:
still have this left over

Reply:
oops sorry yeah.

Reply:
I'll fix these in the morning, literally just saw them now.